.github/workflows/ci.yml

```
@@ -0,0 +1,64 @@
+jobs:
+  test:
+    defaults:
+      run:
+        shell: bash -el {0}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python: ['3.8', '3.9', '3.10', '3.11']
+        include:
+          - os: macos-latest
+            python: '3.10'
+          - os: windows-latest
+            python: '3.10'
```

```suggestion
            python: '3.9'
```

Testing against the oldest version usually reveals more bugs.

---

Done!